This allows us to get some function from math.h that other platforms are
getting from ieeefp.h.
Can send upstream since it already has an ifdef statement for Solaris.

diff --git a/xpcom/ds/nsMathUtils.h b/xpcom/ds/nsMathUtils.h
--- a/xpcom/ds/nsMathUtils.h
+++ b/xpcom/ds/nsMathUtils.h
@@ -13,6 +13,7 @@
 #include <float.h>
 
 #ifdef SOLARIS
+#include <math.h>
 #include <ieeefp.h>
 #endif
 
